frontend/app/Post.tsx

```diff
@@ -1,16 +1,7 @@
 import { GeoPoint } from "firebase/firestore";
+import { Post } from "./interfaces";
+
 
-export interface Post {
```

Comment:
(Non-blocking)
Not sure it was necessary to remove this to create an interfaces file. createPost() is sort of just dangling now... IMO, storing all the objects/interfaces in one file is more bad practice? Especially if you have helper methods...

Reply:
this interface acts as a bridge between Firebase and the rest of the app. Other helper interfaces can be page specific, but in this case its a good idea to keep all firebase specific interfaces in one spot

Reply:
Hmm. Not sure about keeping them all in one spot but yeah, Post.tsx definitely could've been refactored to another folder, i.e., interfaces/, to document that these are firebase interfaces. Then we can add helper methods without having a monolithic file.